modules/oc-mirror-disk-to-mirror.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-disk-to-mirror_{context}"]
+= Mirroring from disk to a mirror in a disconnected environment
+
+// need "in a disconnected environment"?
+
+// TODO: intro
+
+////
+Note: When using oc-mirror lifecycle operations, the above commands are continually reused to create and publish imageset updates as needed. The imageset-config can also be modified between application runs as needed. This promotes oc-mirror usage in workflow automation or cron jobs.
+
+TODO: what exactly does this mean. says that the above commands are continually reused (does this mean automatically, or that users can continually manually reuse them as often as wanted)? Or have to set up their own workflow automation/cron jobs?
+
+TODO: Figure out more on use cases for updating the image set config file before runs - and document that somewhere
+
+////
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`) in the disconnected environment.
+* You have installed the `oc-mirror` CLI plug-in in the disconnected environment.
+* You have created the image set configuration file.
+* You have generated the image set file by using the `oc mirror` command.
+* You have transferred the image set file and the image set configuration file to the disconnected environment.
+// TODO: Confirm prereq about not needing a cluster, but need pull secret misc
+
+// TODO: confirm that they needed to transfer the same image set config file to the disconnected env
```

Comment:
@afflom @jpower432 Can you confirm that we do expect people to transfer that imageset-config.yaml file to the disconnected environment? Or somehow get a similar/the same file there somehow?